Makefile

```diff
@@ -14,10 +14,10 @@ start: start-support
 	@node server.js
 start-nodemon: start-support
 	@yarn nodemon server.js
-start-workspace-host: start-support kill-running-workspaces
-	@node workspace_host/interface.js
-start-workspace-host-nodemon: start-support kill-running-workspaces
-	@yarn nodemon --config workspace_host/nodemon.json workspace_host/interface.js
+workspace-host-start: start-support kill-running-workspaces
+	@yarn workspace-host-start
+workspace-host-dev: start-support kill-running-workspaces
+	@yarn workspace-host-dev
```

Comment:
I'm good with `dev`/`start`. My fingers have a slight preference for `node start-...` because I'm used to the "command verb object" syntax these days (`systemctl start ...`, `git clone ...`, etc) but I don't feel very strongly about this.

Reply:
Sounds good, I can flip the verbs/nouns around.

Reply:
Done in b900b3c / #7581.